scripts/nix-http-export.cgi.in

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
```

Comment:
This is required because line 15/16 are bashism and undefined in sh.

Reply:
Hmm. This might be "wrong" if it's intended for use in NixOS, though? It won't have a /bin/bash and I guess it can assume /bin/sh is from the bash package. 

Also: it hasn't been edited since Feb 2009, so I wonder if this script is still in use. Maybe @7c6f434c knows?

Reply:
This script should be deleted. It's not installed and hasn't been updated since 2009.

Reply:
Done in #5597.

Reply:
It's true that `nix-serve` completely covers all use cases not covered by remote ssh store.

Reply:
rebased and dropped all changes in this file.